### DISCUSSION TITLE
Welcome to quarto-site Discussions!

### DISCUSSION THREAD

**Discussion by JosiahParry** [JosiahParry/quarto-site#4] [Polls]
## 👋 howdy

As you may have noticed, I've been R programming videos mainly focused on spatial analysis. 
I'd like to hear from you what types of videos would be most helpful for you! 
Are spatial analysis videos helpful? Are you getting bogged down by list columns and you'd prefer videos more tailored to R data types? 

I'd like, first and foremost, to learn what type of video format you prefer! 

Do you have a different idea? Let me hear it in the comments :)

**Comment by daranzolin** [comment]
I think you're hitting a sweet spot (and filling a hole) with this 'intermediate' spatial analysis content. There are a gazillion introductions to sf, and a surplus of fairly advanced algorithms that go beyond my skill, but not much for programmers who know some GIS and know some stats.

**Reply by JosiahParry** [reply]
Good to know! Thank you :) 
are there any topics that you'd like content on specifically?
